frontend/views/components/modal/Prompt.vue

```diff
@@ -45,11 +45,11 @@ export default ({
   methods: {
     closeModal () {
       sbp('okTurtles.events/emit', MODAL_RESPONSE, false)
-      this.$refs.modal.close()
+      this.$refs.modal.unload()
     },
     submit () {
       sbp('okTurtles.events/emit', MODAL_RESPONSE, true)
-      this.$refs.modal.close()
+      this.$refs.modal.unload()
```

Comment:
This is not related to the issue itself but it's a bug-fix for an issue I had where once the prompt modal is dismissed the entire page is frozen.